internal/injector/builtin/generator/main.go

```diff
@@ -149,3 +149,38 @@ func writeAll(w io.Writer, data []byte) {
 func init() {
 	log.SetPrefix("github.com/DataDog/orchestrion/internal/injector/builtin/generate: ")
 }
+
+func (g *globs) Set(value string) error {
+	*g = append(*g, value)
+	return nil
```

Comment:
Where is this used?

Reply:
It's implementing the `flag.Value` interface.